source/account_structure/index.html.md.erb

```diff
@@ -1,92 +1,68 @@
 ---
-title: How accounts work
-last_reviewed_on: 2021-01-08
+title: Setting up multiple services
+last_reviewed_on: 2021-02-03
 review_in: 6 months
 weight: 40
 ---
 
-# How accounts work
+# Setting up multiple services
+  
+If you have more than one online service, you can set up your GOV.UK Pay account and payment service provider (PSP) accounts so it's easier to manage your payments and services.
 
-GOV.UK Pay does not limit how many services you can integrate with it and how many accounts you can have with us.
-GOV.UK Pay test and live accounts are free to create and maintain but your payment service provider may charge you for the underlying accounts.
+## Decide where to send users
 
-## Account structure examples
+You can send your users to make payments with either:
 
-If you do integrate multiple services, you can treat them separately or as a
-combined single service within GOV.UK Pay and your payment service provider (PSP).
-This gives 3 possible account configurations.
+- a different GOV.UK Pay service for each of your services
+- a combined GOV.UK Pay service
 
-As an example, take 2 services called ‘parking permits’ and ‘parking fines’:
+### Send your users to different GOV.UK Pay services
 
-### Type 1 (GOV.UK Pay separate, PSP separate)
+Send your users to a different GOV.UK Pay service for each of your services if you need to do any of the following:
 
-<%= image_tag "/images/accountstructure_1.svg", { :alt => '' } %>
+- give each service its own name on the GOV.UK Pay payment pages
+- report on each service's payments separately
+- send each service's payments to different bank accounts
+- give your staff members access to each service separately in the GOV.UK Pay admin tool
 
-The diagram show two services, ‘Parking permits’ and ‘Parking fines’, that are considered separate services by
-both GOV.UK Pay and the PSP.
+To set up your services like this, [create a new GOV.UK Pay service](https://selfservice.payments.service.gov.uk/my-services/create) for each of your online services.
 
-If you use this account structure, you can:
+GOV.UK Pay services are free, and there's no limit to how many you can create.
 
-- give access for each service to different staff members
-- optionally use different service names on the GOV.UK Pay payment pages
-- send each service's payments to a different bank account
-- report separately on each of your services
+To change a service's name, go to the [**My services** page in the GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/my-services).
```

Comment:
happy to keep this here as a reminder but not sure it's directly related?

Reply:
Cool - I've removed this. Think it's a clear journey in the admin tool now we're making My Services the default page.